src/libraries/Microsoft.PowerFx.Interpreter/Values/DelegationRunContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.IR.Nodes;
+
+namespace Microsoft.PowerFx.Types
+{
+    internal readonly struct DelegationRunContext
```

Comment:
>DelegationRunContext [](http://example.com/codeflow?start=29&length=20)

Can this move to PowerFx.Connectors? 

It looks like it's just used for the ODataVisitor?
And rename to ODataVisitorContext.cs?
And add  a parameter

/// Context for the <see=ODataVisitor>

Reply:
yep done